src/sentiment.jl

```diff
@@ -8,7 +8,9 @@ function pad_sequences(l, maxlen=500)
             push!(res, ele)
         end
         return res
-    end
+    else
+        length(1) > maxlen || throw("String length exceeds maximum length")
```

Comment:
`length(1)` => `length(l)`?

Also, why are we checking this condition again? `length(l) > maxlen`

Reply:
Should the checking condition be `&&`? even in that case?